client/src/pages/Scores/Scores.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC } from 'react'
+import { RouteComponentProps } from '@reach/router'
+import { Game } from '../../types/Game'
+import './Scores.scss'
+import Button from '../../components/Button'
+import { GameStatus } from '../../types/GameStatus'
+import { RoundResult } from '../../types/RoundResult'
+import { Player } from '../../types/Player'
+import ScoreBoard from '../../components/ScoreBoard'
+
+export interface ScoresProps extends RouteComponentProps {
+  game: Game
+  currentPlayerId: string
+  isGameHost: boolean
+  result?: RoundResult | undefined
```

Comment:
When defining interfaces, `?` denotes an optionnal property
```suggestion
  result?: RoundResult
```

Reply:
But actually, do we ever display the `<Scores />` component without passing result ?

```suggestion
  result: RoundResult
```

If that's the case, there are a few unecessary checks later in the file.

Reply:
Indeed ✅